mesonbuild/compilers/mixins/intel.py

```diff
@@ -59,6 +59,7 @@ class IntelGnuLikeCompiler(GnuLikeCompiler):
     }  # type: T.Dict[str, T.List[str]]
 
     OPTIM_ARGS = {
+        'plain': [],
```

Comment:
I *think* because it's not annotated, that it gets confused that this is an empty list. adding:
`OPTIM_ARGS: T.Dict[str, T.List[str]] = {` should fix this

Reply:
OPTIM_ARGS isn't commented, only BUILD_ARGS is

Reply:
:woman_facepalming: Thanks. Not sure how I missed it.